app/views/shared/_navbar.html.erb

```diff
@@ -15,16 +15,15 @@
           <%= link_to "All Listings", listings_path, class: "nav-link" %>
         </li>
         <li class="nav-item dropdown">
-          <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
+          <%= image_tag "sakura-photo.jpg", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
           <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
-            <%= link_to "Action", "#", class: "dropdown-item" %>
-            <%= link_to "Another action", "#", class: "dropdown-item" %>
+            <%= link_to profile_path(current_user), class: "dropdown-item" do %>
+              👋 Welcome <%= current_user.first_name %>
+            <% end %>
+            <%= link_to "Bookings", "#", class: "dropdown-item" %>
```

Comment:
Wondering should make the flow for checking all the bookings 
- from the user profile OR
- bookings 
- or both? (in this case, the user profile would only contain summarized info, and bookings will have more detailed info including notes, which a page we don't have yet)
? 
@zuccamia @worksofgenius

Reply:
Checking the user profile seems like the easiest flow. Booking.com and other travel sites seem to use it.

Reply:
I agree with @worksofgenius. I was thinking we can use something like [this](https://jqueryui.com/tooltip/) for the notes and `accept`/`cancel` with this type of js.erb file like [this tutorial](https://flatironschool.com/blog/jquery-and-javascript-meet-erb-embedded-ruby)